dev/com.ibm.ws.cdi.1.2.weld/bnd.bnd

```diff
@@ -1,5 +1,5 @@
 #*******************************************************************************
-# Copyright (c) 2017 IBM Corporation and others.
+# Copyright (c) 2017,2018 IBM Corporation and others.
 # All rights reserved. This program and the accompanying materials
```

Comment:
Needs an additional space between 2017,2018. (There may be more cases.)
2017, 2018